presto-native-execution/presto_cpp/main/functions/tests/FunctionMetadataTest.cpp

```diff
@@ -40,24 +40,48 @@ class FunctionMetadataTest : public ::testing::Test {
     functionMetadata_ = getFunctionsMetadata();
   }
 
+  static std::string getFunctionKindAndParams(const json& entry) {
+    auto str = entry["functionKind"].get<std::string>();
+    if (entry["paramTypes"].is_array()) {
+      for (auto const& paramType : entry["paramTypes"]) {
+        str += "-" + paramType.get<std::string>();
+      }
+    }
+    return str;
+  }
+
+  static void sortMetadataList(json::array_t& list) {
```

Comment:
Why did you make this method static ? Doesn't seem like there is any need for it to be so.

Reply:
I made it static because this member function doesn't need to access the `this` pointer. This way the function can be used by other static methods if needed. However, you're right, that this function can be made non-static, because the caller is also non-static and there is no other callers.